ts/test/test-profiler.ts

```diff
@@ -383,6 +380,25 @@ describe('Profiler', () => {
          assert.ok(
              requestProfileMock.isDone(), 'expected call to create profile');
        });
+    it('should receive heap profile response without error.', async () => {
```

Comment:
Drop trailing full stop in the test description here and in all other places where you have it. See https://jasmine.github.io/2.1/introduction.html, these do not have the full stops.

Reply:
Fixed, at least within modified and new tests here.